Library/Homebrew/utils/github.rb

```diff
@@ -501,6 +500,26 @@ def check_for_duplicate_pull_requests(name, tap_remote_repo, state:, file:, args
     end
   end
 
+  def get_long_build_pull_requests_files(name, tap_remote_repo)
```

Comment:
This doesn't feel right to be in this class; it's way too specific to `brew pull`.

Reply:
I moved this out of the GitHub file